docs/standard/guidance-architecture.md

```diff
@@ -24,4 +24,8 @@ This guide is an introduction to the recommended architecture, design, and deplo
 
 ## [Architecting Container and Microservice Based Applications](microservices-architecture/index.md)
 
-This guide is an introduction to developing microservices-based applications and managing them using containers. It discusses architectural design and implementation approaches using .NET Core and Docker containers. 
+This guide is an introduction to developing microservices-based applications and managing them using containers. It discusses architectural design and implementation approaches using .NET Core and Docker containers.
+
+## [Serverless Apps. Architecture, Patterns, and Azure Implementation](serverless-architecture/index.md)
```

Comment:
We use sentence case for all headers. Here, it would be "Serverless apps architecture, patterns, and Azure implementation"

Reply:
Even for the book title?

Reply:
Yes, we'd use sentence case in all titles and headers.